patches/@oclif+dev-cli+1.26.0.patch

```diff
@@ -0,0 +1,82 @@
+diff --git a/node_modules/@oclif/dev-cli/lib/commands/pack/index.js b/node_modules/@oclif/dev-cli/lib/commands/pack/index.js
```

Comment:
This should be generated at `packages/cli/patches` , aka one level down, and not at root of the repo.

This means that when we `npm i` in each subpackage, the patch will only be applied there.

Reply:
Fixed, resolving convo